app/controllers/speedrun_dot_com_users_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SpeedrunDotComUsersController < ApplicationController
+  def create
+    if SpeedrunDotComUser.from_api_key!(params[:speedrun_dot_com_user][:api_key].strip, user: current_user)
```

Comment:
Should we force logins on these actions so that `current_user` will never be nil?